package-lock.json

```diff
@@ -144,11 +144,11 @@
         "@pixi/spritesheet": "7.1.0-alpha",
```

Comment:
I'm not sure if I regenerated the lock file correctly? seems to have more changes than I expected. I just ran `npm install` in the top level after updating the `package.json` for node

Reply:
I tried `npm i gl@6.0.1 @types/gl@6.0.1` in `bundles/pixi.js-node` and got the same result, so I think it is OK. Possibly it is caused by the dependency change of `gl`.

Reply:
@bigtimebuddy by the way, should we have a `npm audit fix` for 7.1.0?

Reply:
Yeah, we tried an audit awhile ago (#8480), but kept being blocked by various things.  I'm open to doing this again.